src/Utilities.cc

```diff
@@ -110,15 +110,31 @@ namespace ChimeraTK::Utilities {
 
   /********************************************************************************************************************/
 
+  // raises a logic error if the name has multiple consecutive slashes
+  void raiseIfdoublelash(const std::string& name) {
+    if(name.find("//") != std::string::npos) {
+      throw ChimeraTK::logic_error(name + " variable names cannot contain consecutive slashes");
+    }
+  }
+
+  /********************************************************************************************************************/
+
+  // strip trailing slashes unless the name consists of a single slash
+  // will raise a logic_error, if there are consecutive slashes
+  // Uses to unify module names
   std::string stripTrailingSlashes(const std::string& name) {
-    if(name.empty() || name[name.size() - 1] != '/') return name;
+    if(name.size() <= 1 || name[name.size() - 1] != '/') return name;
+    raiseIfdoublelash(name);
     return stripTrailingSlashes(name.substr(0, name.size() - 1));
   }
```

Comment:
I understood from the discussion this morning that we do not want to allow trailing slashes at all (except the name is just "/", i.e. size() == 1, and it's a module, not a variable). Did I get this wrong?